tests/tests_ntp.yml

```diff
@@ -11,6 +11,8 @@
         iburst: yes
         minpoll: 4
         maxpoll: 6
+    timesync_ntp_custom_settings:
+      - "tinker panic 0"
```

Comment:
The test should check if this setting was applied correctly

Reply:
I'm not sure how to check this specific setting by inspecting runtime state, the closest I can come up with is looking for this line in the generated config file.  Is that enough, or are you thinking something more?

Reply:
> I'm not sure how to check this specific setting by inspecting runtime state, the closest I can come up with is looking for this line in the generated config file. Is that enough, or are you thinking something more?

I'll defer to @mlichvar about that - but IMO it is better to check the generated config file than to do nothing.

Reply:
There might be a possibility to do that with `ntpq`, but it would need authentication to be set up. I think it's better to check the generated file.